Documentation/observability/metrics.rst

```diff
@@ -175,6 +175,39 @@ OpenMetrics imposes a few additional requirements on metrics names and labels,
 so this functionality is currently opt-in, though we believe all of the Hubble
 metrics conform to the OpenMetrics requirements.
 
+
+Cluster Mesh API Server Metrics
+===============================
+
+Cluster Mesh API Server metrics provide insights into both the state of the
+``clustermesh-apiserver`` process and the sidecar etcd instance.
+Cluster Mesh API Server metrics are exported under the ``cilium_`` Prometheus
+namespace. Etcd metrics are exported under the ``etcd_`` Prometheus namespace.
```

Comment:
Are we fine with *Cluster Mesh API Server metrics are exported under the ``cilium_`` Prometheus
namespace*, or should we expose them under the `clustermesh_apiserver` namespace?

Reply:
They don't use both, like the operator does (i.e. `cilium_clustermesh_apiserver_`)?

Reply:
Currently not, because this PR is exposing a few etcd metrics which were already present in cilium (using the same format). But I can make the `Namespace` part configurable and change it for the clustermesh-apiserver.